syntax = "proto3";

package ca_simulator;

service SimEngine {
    rpc RunSimulationGif (SimRequest) returns (SimReply) {}
}

message SimRequest {
    string params = 1;
}

message SimReply { 
    string simulation = 1;
}